tests/regression_tests/source_parameterized_dlopen/inputs_true.dat

```diff
@@ -18,6 +18,6 @@
     <particles>1000</particles>
     <batches>10</batches>
     <inactive>0</inactive>
-    <source library="build/libsource.so" parameters="1e3" strength="1.0"/>
+    <source strength="1.0" type="source"/>
```

Comment:
This source should still have a `'library'` member, if I understand the changes to `source.py` correctly.